debezium-core/src/main/java/io/debezium/jdbc/JdbcConnection.java

```diff
@@ -515,6 +521,36 @@ public synchronized void close() throws SQLException {
     }
 
     /**
+     * Returns a JDBC connection string using the current configuration and url. 
+     * 
+     * @param urlPattern a {@code String} representing a JDBC connection with variables that will be replaced
+     * @return a {@code String} where the variables in {@code urlPattern} are replaced with values from the configuration
+     */
+    public String connectionString(String urlPattern) {
+        Properties props = config.asProperties();
+        return findAndReplace(urlPattern, props, JdbcConfiguration.DATABASE, JdbcConfiguration.HOSTNAME, JdbcConfiguration.PORT,
+                              JdbcConfiguration.USER, JdbcConfiguration.PASSWORD);   
+    }
```

Comment:
I really wonder if we should include the password value, especially if this is used for logging. IMO the password should never appear in the log.